modules/nf-core/mageck/count/tests/main.nf.test

```diff
@@ -0,0 +1,69 @@
+// TODO nf-core: Once you have added the required tests, please run the following command to build this file:
+// nf-core modules test mageck/count
+nextflow_process {
+
+    name "Test Process MAGECK_COUNT"
+    script "../main.nf"
+    process "MAGECK_COUNT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "mageck"
+    tag "mageck/count"
+
+    test("mageck-count - fastq") {
+
+        config "./nextflow.config"
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test,test2', single_end:true] , // meta map
```

Comment:
You might want to adjust the filepaths to look like this:
`file(params.modules_testdata_base_path + 'genomics/mus_musculus/mageck/count_table.csv', checkIfExists: true)`
But lets solve the other issues first